bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -1947,6 +1947,181 @@
 	</channel-type>
 
 
+	<channel-type id="channel232" advanced="true">
+		<item-type>Number</item-type>
+		<label>Channel 232</label>
+		<state readOnly="true"/>
+	</channel-type>
```

Comment:
Maybe create only one channel type for all unknown channels.

Reply:
I'm not sure what exactly needs to be changed here. Could you explain that a bit more?

Reply:
Rather having many channel types with a number, you could just define a channel type for unknown data with a description explaining the situation, like for example:
```
	<channel-type id="unknownChannel" advanced="true">
		<item-type>Number</item-type>
		<label>Unknown Channel</label>
		<description>blablabla</description>
		<state readOnly="true"/>
	</channel-type>
```

Reply:
tried to change that accordingly. Hope that is what you were referring to.

Reply:
Yes, exactly.
But as you want a different channel label for each channel, something is missing.
I am going to push a PR to your repo.